<?xml version='1.0' encoding='UTF-8' standalone='no'?>
<doxygen xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="compound.xsd" version="1.8.13">
  <compounddef id="structxn_1_1ExceededMaxIterations" kind="struct" language="C++" prot="public">
    <compoundname>xn::ExceededMaxIterations</compoundname>
    <basecompoundref refid="structxn_1_1XNetworkException" prot="public" virt="non-virtual">xn::XNetworkException</basecompoundref>
    <includes refid="exception_8hpp" local="no">exception.hpp</includes>
      <sectiondef kind="public-func">
      <memberdef kind="function" id="structxn_1_1ExceededMaxIterations_1a6c7b18af1fb04b89d8a00be8a670ef92" prot="public" static="no" const="no" explicit="yes" inline="yes" virt="non-virtual">
        <type></type>
        <definition>xn::ExceededMaxIterations::ExceededMaxIterations</definition>
        <argsstring>(const char *msg)</argsstring>
        <name>ExceededMaxIterations</name>
        <param>
          <type>const char *</type>
          <declname>msg</declname>
        </param>
        <briefdescription>
        </briefdescription>
        <detaileddescription>
        </detaileddescription>
        <inbodydescription>
        </inbodydescription>
        <location file="/media/lubuntu/USBDISK/github/ckpttncpp/lib/include/xnetwork/exception.hpp" line="176" column="1" bodyfile="/media/lubuntu/USBDISK/github/ckpttncpp/lib/include/xnetwork/exception.hpp" bodystart="176" bodyend="179"/>
      </memberdef>
      </sectiondef>
    <briefdescription>
    </briefdescription>
    <detaileddescription>
<para>Raised if (a loop iterates too many times without breaking. This may occur, for example, in an algorithm that computes progressively better approximations to a value but exceeds an iteration bound specified by the user. </para>    </detaileddescription>
    <inheritancegraph>
      <node id="27">
        <label>xn::XNetworkException</label>
        <link refid="structxn_1_1XNetworkException"/>
        <childnode refid="28" relation="public-inheritance">
        </childnode>
      </node>
      <node id="28">
        <label>std::runtime_error</label>
      </node>
      <node id="26">
        <label>xn::ExceededMaxIterations</label>
        <link refid="structxn_1_1ExceededMaxIterations"/>
        <childnode refid="27" relation="public-inheritance">
        </childnode>
      </node>
    </inheritancegraph>
    <collaborationgraph>
      <node id="30">
        <label>xn::XNetworkException</label>
        <link refid="structxn_1_1XNetworkException"/>
        <childnode refid="31" relation="public-inheritance">
        </childnode>
      </node>
      <node id="31">
        <label>std::runtime_error</label>
      </node>
      <node id="29">
        <label>xn::ExceededMaxIterations</label>
        <link refid="structxn_1_1ExceededMaxIterations"/>
        <childnode refid="30" relation="public-inheritance">
        </childnode>
      </node>
    </collaborationgraph>
    <location file="/media/lubuntu/USBDISK/github/ckpttncpp/lib/include/xnetwork/exception.hpp" line="175" column="1" bodyfile="/media/lubuntu/USBDISK/github/ckpttncpp/lib/include/xnetwork/exception.hpp" bodystart="174" bodyend="180"/>
    <listofallmembers>
      <member refid="structxn_1_1ExceededMaxIterations_1a6c7b18af1fb04b89d8a00be8a670ef92" prot="public" virt="non-virtual"><scope>xn::ExceededMaxIterations</scope><name>ExceededMaxIterations</name></member>
      <member refid="structxn_1_1XNetworkException_1adf7ac46d6cef6d2fcee880fc2f709458" prot="public" virt="non-virtual"><scope>xn::ExceededMaxIterations</scope><name>XNetworkException</name></member>
    </listofallmembers>
  </compounddef>
</doxygen>
